packages/governance/test/swingsetTests/contractGovernor/bootstrap.js

```diff
@@ -163,6 +163,27 @@ const setupElectorateChange = async (
   return { details, outcomeOfUpdate };
 };
 
+const setupApiCall = async (zoe, log, governor, installations) => {
+  const { details, instance, outcomeOfUpdate } = await E(
+    governor,
+  ).voteOnApiInvocation(
+    'governanceApi',
+    [], // empty params
+    installations.binaryVoteCounter,
+    2n,
+  );
+
+  E(E(zoe).getPublicFacet(instance))
+    .getOutcome()
+    .then(outcome => log(`vote outcome: ${q(outcome)}`))
+    .catch(e => log(`vote failed ${e}`));
```

Comment:
tangential, can/should our log distinguish between error messages and others? (likd stdout stderr)